### ISSUE TITLE
range data format

### ISSUE THREAD

**Issue by Enri2077** [aremazeilles/eurobench_documentation#86]
Hi @aremazeilles,

I'm working on beast and need a data format representing a distance through time.
I considered using the `Joint center 3D trajectory` or `Landmark 3D trajectory` data formats, reporting a position in meters through time, but having the x, y and z fields is not ideal since I need to represent a distance rather than a position.

The data is computed as the minimum range between the trolley and obstacles at any time, measured with a lidar/range finder.
I suggest the following data format:
time | range
------------ | -------------
| `sec` | `m` |

What do you think?

**Comment by aremazeilles** [comment]
ping @alfonsotecnalia 

I see two options:

* Create a specific format, `distance.csv`
* Use the generic format `platformData.csv` since it can contain any information.

The advantage of the first option is that we know from the file name its content. I don't mind adding such format.
If so , would it make sense to suggest a generic description, such as:

* if a single distance is recorded, the col label could be `distance`.
* If several distances are recorded, the col label should be extended with a landmark name: `distance_name1`, `distance_name2`


I would rather use the `distance` term rather than `range`, as it describes better the information stored.
Any opinion?

**Comment by Enri2077** [comment]
I'm perfectly ok with the distance format, including the option to include multiple columns! I'll update my data format.

**Comment by alfonsotecnalia** [comment]
Ok with including distance format.

**Comment by aremazeilles** [comment]
@Enri2077 did you use that format at the end? If yes, please point me to your files, and I will adjust documentation accordingly

**Comment by Enri2077** [comment]
Yes I used it. I still don't have a definitive dataset, but I'll attach a file using this format: 
[subject_001_cond_007_run_001_distance.csv.zip](https://github.com/aremazeilles/eurobench_documentation/files/6100668/subject_001_cond_007_run_001_distance.csv.zip)

**Comment by aremazeilles** [comment]
Would #88 make the job @Enri2077 ?

**Comment by Enri2077** [comment]
Perfect, thanks!
